bin/gaia-marionette

```diff
@@ -64,6 +64,7 @@ if [ -z "$TEST_FILES" ]; then
       if [ -z "$HARNESS_ONLY" ]; then
         TEST_FILES=$(find $DIR/.. -path "*/apps/*/test/marionette/*_test.js" \
                               -or -path "*/dev_apps/*/test/marionette/*_test.js" \
+                              -or -path "*/tv_apps/*/test/marionette/*_test.js" \
```

Comment:
In this script we are simply pulling out all integration test files, tests will be filtered in the runtime when it knows which device runs on. 

And this is because some features will be tested on more than one device types, and it can't simply filter tests by folder name, so we add an options to mocha test to declare target device type and mocha will do that for us. 

Here's an example: https://github.com/mozilla-b2g/gaia/blob/master/tv_apps/smart-system/test/marionette/browser_context_menu_test.js#L53